website/www/site/content/en/documentation/ml/preprocess-data.md

```diff
@@ -45,18 +39,23 @@ modules for machine learning tasks.
     -   Count the occurrences of words in all the documents to calculate
         [TF-IDF](https://en.wikipedia.org/wiki/Tf%E2%80%93idf)
         weights.
+    -  Generate [embeddings](https://en.wikipedia.org/wiki/Embedding) on text data using large language models (LLMs).
```

Comment:
We might want to make this the first list item, depending on how much we want to emphasize embeddings. We could also move the Text embedding transforms section before the Data processing transforms that use TFT section.